pkg/k8s/watchers/pod.go

```diff
@@ -217,12 +217,15 @@ func (k *K8sWatcher) addK8sPodV1(pod *slim_corev1.Pod) error {
 		return k.deleteK8sPodV1(pod)
 	}
 
-	skipped := false
+	if pod.Spec.HostNetwork {
+		logger.Debug("pod is using host networking")
```

Comment:
Now that this is a log msg, maybe we can elaborate a bit more on it? I think we should at the least conform to other log msgs so capitalize the first word

Reply:
@tklauser if you get a chance, pls follow-up if needed once back

Reply:
@christarazi as far as I can see, the `logger` already include necessary additional information as fields:

https://github.com/cilium/cilium/blob/b875c1d61ba8e52c5a7e31d9f25165a4a9916d3e/pkg/k8s/watchers/pod.go#L202-L208

Or do you have any other info in mind that would be relevant for this case? Otherwise I'd just change the capitalization of the log message.

Reply:
Capitalization is what I had in mind. If that's the only change you're going to make, then it's not worth making a PR for it; only if you have other changes already planned

Reply:
Thanks. I have some other changes for the same package, so I can fold in the capitalization change.